cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpCache.java

```diff
@@ -0,0 +1,29 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import okhttp3.Cache;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public final class CwmsHttpCache {
+    public static final String CACHE_SIZE_PROPERTY_KEY = "cwms.http.client.cache.size";
```

Comment:
I'm trying to decide if this class should have its own builder?  I like the way you did the cacheDir and cacheSize from properties with defaults but I'm wondering if that should just be the initial values and if people wanted to they could build a CwmsHttpCache via new CwmsHttpCache.Builder().withCacheDir("somedir").withCacheSize(50*MB).build()  sort of thing?

Reply:
Looking at it again, this class doesn't build a CwmsHttpCache, it builds a Cache.  So this class sort of serves two functions - a place to hold the -D.  And a class to hold a static instance.  Holding those -D values is equivalent to the private fields in a Builder instance.  I'm not sure a static Cache instance matters.  I think the class name threw me, I'm now thinking that maybe the meat of this class should just move into OkHttpClientInstance